hibernate-core/src/test/java/org/hibernate/jpa/test/criteria/basic/AggregationResultTest.java

```diff
@@ -88,22 +72,6 @@ public void testSumOfIntegers() {
 		em.close();
 	}
 
-	/**
-	 * Sum of Doubles should return a Double
-	 */
-	@Test
-	public void testSumOfDoubles() {
-		EntityManager em = getOrCreateEntityManager();
-		em.getTransaction().begin();
-		CriteriaQuery<Double> criteria = builder.createQuery( Double.class );
-		Root<Product> productRoot = criteria.from( Product.class );
-		criteria.select( builder.sum( productRoot.get( Product_.price ) ) );
-		Object sumResult = em.createQuery( criteria ).getSingleResult();
-		assertReturnType( Double.class, sumResult );
-		em.getTransaction().commit();
-		em.close();
-	}
-
 	/**
 	 * Sum of Floats should return a Float; note that this is distinctly different than JPAQL
```

Comment:
This testing will fail because of the code change by you to return double type as opposed to float (An exception will be thrown complaining returning type is not expected). Not sure what to do then.